website/content/en/preview/concepts/provisioners.md

```diff
@@ -298,7 +301,7 @@ spec:
 
```

Comment:
Do you mind removing the "AWS" section as part of this PR? It's no longer necessary and we should just rip it out now since you are changing this section.